ddocs/medic-client/views/doc_summaries_by_id/map.js

```diff
@@ -1,48 +1,39 @@
 function(doc) {
-  var getPlaceHierarchy = function(contact) {
-    if (!contact) {
-      return;
-    }
-    var place = contact.parent;
+  var getLineage = function(contact) {
     var parts = [];
-    while (place) {
-      if (place.name) {
-        parts.push(place.name);
+    while (contact) {
+      if (contact._id) {
+        parts.push(contact._id);
       }
-      place = place.parent;
+      contact = contact.parent;
     }
     return parts;
   };
 
   if (doc.type === 'data_record' && doc.form) { // report
-    var from = (doc.contact && doc.contact.name) ||
-               doc.from ||
-               doc.sent_by;
-    emit([ doc._id ], {
+    emit(doc._id, {
       _rev: doc._rev,
-      from: from,
+      from: doc.from || doc.sent_by,
       phone: doc.contact && doc.contact.phone,
       form: doc.form,
       read: doc.read,
       valid: !doc.errors || !doc.errors.length,
       verified: doc.verified,
       reported_date: doc.reported_date,
-      place: getPlaceHierarchy(doc.contact)
+      contact: doc.contact && doc.contact._id,
+      lineage: getLineage(doc.contact && doc.contact.parent)
```

Comment:
> that sets a precedent we're not following anywhere else

I get that point. I don't think it's necessary everywhere though. This is specifically the whole-doc-obj vs. just-id-of-doc, it's something that I bump into regularly and that slows me down, I end up having to read the whole file or run the code to figure out which is which. It's a bit of a pet peeve.

@mandric I disagree that external documentation can make up for imprecise naming. Code should be as clear as possible. Two or three extra characters in a name are totally worth it if they reduce my cognitive load.
I do agree that types will help (since in this instance I'm basically asking to document type in the name...)

Gareth I can disagree and commit on this, just leave it to `lineage` if you want.